0000-template.md

```diff
@@ -1,15 +1,22 @@
 # Feature name
 
 * Proposal: [SE-NNNN](NNNN-filename.md)
-* Author: [Swift Developer](https://github.com/swiftdev)
+* Authors: [Author 1](https://github.com/swiftdev), [Author 2](https://github.com/swiftdev)
```

Comment:
The problem can already be seen in #499 (e.g. "Authors" and "Bugs" in [SE-0011](https://github.com/addisonleong/swift-evolution/blob/3abbed3edd12dd21061181993df7952665d660dd/proposals/0011-replace-typealias-associated.md)).